include/aws/testing/aws_test_harness.h

```diff
@@ -322,6 +329,7 @@ static int aws_run_test_case(struct aws_test_harness *harness) {
 
 #define AWS_RUN_TEST_CASES(...)                                                                                        \
     struct aws_test_harness *tests[] = { __VA_ARGS__ };                                                                \
+    aws_mutex_init(&allocator_lock, aws_default_allocator());                                                          \
```

Comment:
We should probably have a static initializer macro. Both pthreads and windows mutexes/SRW locks support this

Reply:
Yeah that allocator isn’t actually necessary anyways IIRC